build/RepositoryBuild.targets

```diff
@@ -84,4 +94,26 @@
     <Message Text="============ Done building $(RepositoryToBuild) ============" Importance="High" />
   </Target>
 
+  <Target Name="_TestRepository" DependsOnTargets="GetRepoBuildProps">
+    <PropertyGroup>
+      <BuildArguments>/t:Test /p:NoBuild=$(_NoBuildRepos) $(RepositoryBuildArguments)</BuildArguments>
+
+      <_BuildScriptToExecute Condition="'$(OS)'!='Windows_NT'">build.sh</_BuildScriptToExecute>
+      <_BuildScriptToExecute Condition="'$(OS)'=='Windows_NT'">build.cmd</_BuildScriptToExecute>
+    </PropertyGroup>
+
+    <Message Text="============ Testing $(RepositoryToBuild) ============" Importance="High" />
```

Comment:
Do we have to copy? Doesn't Korebuild have an argument which repo to build?

Reply:
+ Seems to be duplicated

Reply:
No, KoreBuild does not have an argument that allows you to build one repo. It would if we did #739

Reply:
Updated to reduce some duplication between these targets